03-react/02-users-list/src/main.tsx

```diff
@@ -0,0 +1,283 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import '../style.css';
+
+const appDiv = document.getElementById("app");
+
+if (!(appDiv instanceof HTMLDivElement)) {
+	throw new Error("No div with id 'app' found");
+}
+
+interface UserInfo {
+	id: number;
+	avatar: string;
+	first_name: string;
+	last_name: string;
+	age: number;
+	city: string;
+	ip: string;
+	isAdmin: boolean;
+
+};
+
+interface UserProps {
+	User: UserInfo
```

Comment:
Another small thing I didn't see before: by convention, we use lower-case names for objects properties. So that would be `user` here. (However, the function/component `User()` should stay as-is.)